modules/client/examples/06-multisig-client/00-installation.ts

```diff
@@ -0,0 +1,73 @@
+/* MARKDOWN
+## Multisig governance plugin client
+### Creating a DAO with a multisig plugin
+*/
+import {
+  Client,
+  Context,
+  DaoCreationSteps,
+  GasFeeEstimation,
+  ICreateParams,
+  MultisigPluginInstallParams,
+} from "@aragon/sdk-client";
+import { MultisigClient } from "../../src";
+import { contextParams } from "../00-client/00-context";
+
+const context: Context = new Context(contextParams);
+const client: Client = new Client(context);
+
+// Define the plugins to install and their params
+
+const members: string[] = [
+  "0x1234567890123456789012345678901234567890",
+  "0x2345678901234567890123456789012345678901",
+  "0x3456789012345678901234567890123456789012",
+  "0x4567890123456789012345678901234567890123",
+];
+
+const multisigIntallParams: MultisigPluginInstallParams = {
+  minApprovals: BigInt(3),
```

Comment:
Does this need to be a bigint? IMO it should be a regular number, and only convert it to a `bigint` internally if needed

Reply:
Yeah, you are right. I hope there isnt a multisig with more than 9007199254740992 memebers